api-test/src/test/java/org/openmrs/module/metadatasharing/util/MetadataUtilTest.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.metadatasharing.util;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.openmrs.api.context.Context;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+public class MetadataUtilTest extends BaseModuleContextSensitiveTest {
+	
+	@Test
+	public void testSetupStandardMetadata() throws Exception {
+		assertThat(Context.getVisitService().getVisitTypeByUuid("86b3d7bc-d91f-4ce2-991c-f71bba0b31e4"), nullValue());
+		
+		boolean anyChanges = MetadataUtil.setupStandardMetadata(getClass().getClassLoader());
+		
+		assertTrue(anyChanges);
+		assertThat(Context.getVisitService().getVisitTypeByUuid("86b3d7bc-d91f-4ce2-991c-f71bba0b31e4").getName(),
+		    is("Clinic or Hospital Visit"));
+	}
+	
+	@Test
+	public void testSetupStandardMetadataFromAliasedPackageFile() throws Exception {
+		String aliasedPackage = "packages2.xml";

```

Comment:
Give this file a better name than packages2.xml. Call it packages-aliased or something.